### DISCUSSION TITLE
Calculation of adsorption energy using pyscf

### DISCUSSION THREAD

**Discussion by rudin-jiang** [pyscf/pyscf#2645] [Q&A] [answered]
I have used VASP before to calculate the adsorption energy of small molecules on solid surfaces, which are modelled using a slab model. In vasp I used pbe functional, paw pseudopotential, and the cut off energy was 400 eV. now I want to do a similar calculation using pyscf, and I would like to ask what would be the reasonable settings for Cell.basis, Cell.ecp, and Cell.pseudo in this type of calculation.

**Comment by tberkel** [comment]
Unfortunately there is no simple answer to this question. You should do
your own testing whenever possible. Moreover, robust support of good
pseudopotentials and associated basis sets for solids containing heavy
atoms (third row and beyond) is currently lacking in PySCF.

**Comment by rudin-jiang** [comment, marked as answer]
Thank you!

I found a site that mentioned in general cases, using the DZVP-MOLOPT-SR-GTH basis set and the GTH-PBE pseudopotential for geometry optimization is appropriate, while the TZV2P-MOLOPT-GTH basis set is more accurate for energy calculations. I am planning to do some comparative tests to verify these viewpoints.

http://bbs.keinsci.com/thread-23361-1-1.html
